dtests/src/test/java/io/snappydata/hydra/udfs/JavaUDF3.java

```diff
@@ -0,0 +1,36 @@
+package io.snappydata.hydra.udfs;
+
+import org.apache.spark.sql.api.java.UDF3;
+import scala.collection.mutable.WrappedArray;
+
+public class JavaUDF3 implements UDF3<WrappedArray<String>, WrappedArray<String>, WrappedArray<String>, String> {
```

Comment:
Please add descriptions for the functions.

Reply:
Done.